llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -1330,7 +1330,7 @@ void RISCVInstructionSelector::emitFence(AtomicOrdering FenceOrdering,
 namespace llvm {
 InstructionSelector *
 createRISCVInstructionSelector(const RISCVTargetMachine &TM,
-                               RISCVSubtarget &Subtarget,
+                               const RISCVSubtarget &Subtarget,
```

Comment:
Please commit this directly independent of this patch.

Reply:
Make RBI const too.

The targets inconsistent about whether these arguments are const reference or not. Maybe we should fix them all?

Reply:
I am stacking this patch on https://github.com/llvm/llvm-project/pull/98243.